index.js

```diff
@@ -1,3 +1,4 @@
+import 'core-js'
```

Comment:
Is there a reason we really need that? I required fromEntries aswell today but used reduce instead.

Shouldn't the bable depedency take care of all polyfills?

Reply:
It only takes care of some polyfills, I guess... It's not really transparent to me, which ones though... Seems like it doesn't polyfill proposals...

I removed the `core-js` dependency again...